pkg/lang/ir/v1/r.go

```diff
@@ -18,12 +18,16 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/cockroachdb/errors"
 	"github.com/moby/buildkit/client/llb"
 )
 
-func (g generalGraph) installRLang(root llb.State) (llb.State, error) {
-	return llb.State{}, errors.New("not implemented")
+func (g generalGraph) installRLang(root llb.State) llb.State {
+
+	installR := "apt-get update && apt-get install -y -t focal-cran40 r-base"
+
+	run := root.Run(llb.Shlexf("bash -c \"%s\"", installR),
+		llb.WithCustomNamef("[internal] installing r-base packages"))
```

Comment:
>r-base packages

This seems ambiguous. Perhaps "r-base APT package"?

Reply:
Sorry for the confusion. The package actually comes from CRAN official repo so I think calling it `apt install R environment from CRAN repository` would be better.